tests/performance/mmap_vs_directio/wikimedia.sd

```diff
@@ -0,0 +1,24 @@
+schema wikimedia {
+
+  document wikimedia {
+    field title type string {
+      indexing: index | summary
+      index: enable-bm25
+    }
+    field text type string {
+      indexing: index
+      index: enable-bm25
+    }
+  }
+
+  fieldset default {
+    fields: title
```

Comment:
We should also include the `text` field to ensure we search in both indexes.

Reply:
Fixed